src/modules/polls.ts

```diff
@@ -233,28 +229,30 @@ export async function handleCommand(
 			try {
 				const id = interaction.options.getString("id", true);
 				const title = interaction.options.getString("title", true);
-				const cron = interaction.options.getString("cron", true);
 				const channel = interaction.options.getChannel("channel", true);
+				const cron = interaction.options.getString(
+					"schedule",
+					false
+				) as string | null;
 
-				// TODO: don't take this at face value
```

Comment:
Why'd you remove this? You can use `cron.validate`, but there should be some extra checks to make sure it doesn't run every second or something

Reply:
Will implement checks in upcoming improvements PR